### PR TITLE
Added a11y features, buttons, and updated radio button

### PR DESCRIPTION
**Pull request by nishatreddy**
- added accessibility features to any component that requires interaction and needs to be read by the screen reader
- added microphone button on the home page for speech recognition @paytonshaltis did the frontend part of it, lmk if there's anything else you need
- added delete goon card account button on the settings page with confirmation message
- fixed the spacing issues for the radio buttons on the settings page
- changed the order of the toggle -> label to label -> toggle

### CODE REVIEW

File: views/account/settings/index.ejs [hunk 1]
```diff
@@ -4,7 +4,16 @@
     <%- include('../../partials/header'); %>
     <title>Settings</title>
     <link rel="stylesheet" href="/account/settings/style" />
-    <script src="/account/settings/main" type="module"></script>
+    <!-- <script src="/main.js" type="module"></script> -->
```
Comment (sderacy) [inline views/account/settings/index.ejs:7]: Why was this changed to line 8?

File: views/account/settings/index.ejs [hunk 2]
```diff
@@ -136,17 +156,31 @@
                   name="theme"
                   class="form-control"
                   id="editTheme"
+                  aria-label="theme"
                 ></select>
               </div>
             </div>
           </div>
+          <br />
```
Comment (sderacy) [inline views/account/settings/index.ejs:164]: 2 things: 

- I'd like to see if there is a way to have the confirm prompt not within the onclick attribute
- I think the button should go after the save changes and cancel changes buttons

File: views/partials/navbar.ejs [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <header class="mb-auto">
   <nav id="nav" class="px-3 navbar navbar-expand-lg navbar-dark">
-    <a id="logo-title" class="navbar-brand text-warning contrast" href="/">G.O.O.N.</a>
+    <a id="logo-title" class="navbar-brand text-warning contrast" href="/"
```
Comment (sderacy) [inline views/partials/navbar.ejs:3]: I'm ok with reformatting it to look prettier but the end of the bracket should stay on line 3 and </a> should all be on one line

**Review by sderacy — CHANGES_REQUESTED** [review]
Looks good so far! Just want a couple of things looked at before we merge this though.

File: views/account/settings/index.ejs (hunk 1 above)
Comment (thadakn1) [reply views/account/settings/index.ejs:7]: i was using this approach to debug the issue of why this ejs file isn't connecting with the main.js file. i'll change it back for consistency purposes.

**Comment by sderacy** [comment]
I also noticed this through testing:
image

I'm not sure if this was an issue before but I'm gonna look into it.

File: views/account/settings/index.ejs (hunk 2 above)
Comment (thadakn1) [reply views/account/settings/index.ejs:164]: - i was looking into it for like hours and i still don't understand why it's just not pulling from the main.js file. i can still keep working on it because i really want all logic/JS to be in the main.js file. 

- got it, im fine with that too! will do it right away :)

**Comment by thadakn1** [comment]
also another concern I had, the save changes and cancel buttons kept getting squished together with no spacing. i tried adding a div, a break statement, and even played with the css but i think the reason the issue persists is because it's within the form element. in one of my million attempts to change it, i think one solution was to move the buttons outside of the form and it formatted better, but unfortunately, moving the buttons outside of the <form> element made it unfunctional and the changed settings wouldn't be saved.
Screen Shot 2022-11-14 at 2 37 48 PM

**Comment by thadakn1** [comment]
> I also noticed this through testing (the text labels by the toggle): image
> 
> I'm not sure if this was an issue before but I'm gonna look into it.

i think even in the past, i noticed that the alignment of the toggle and the label was a bit off. but with the contrast feature enabled, its much more noticable

File: views/partials/navbar.ejs (hunk 3 above)
Comment (thadakn1) [reply views/partials/navbar.ejs:3]: so unfortunately, prettier is what keeps changing it i think. every time i save the file, it formats it so that it goes back to a different line.

File: views/account/settings/index.ejs (hunk 1 above)
Comment (sderacy) [reply views/account/settings/index.ejs]: No worries! I fixed it already

File: views/partials/navbar.ejs (hunk 3 above)
Comment (sderacy) [reply views/partials/navbar.ejs:3]: Weird. We'll leave it alone I guess

File: views/account/settings/index.ejs (hunk 2 above)
Comment (sderacy) [reply views/account/settings/index.ejs:164]: You can fix the button placement, I'll look into the button.

**Comment by sderacy** [comment]
Also I forgot to say this earlier, but I really appreciate that you put alt tags on everything. That's huge. Nice work with that :)

**Comment by sderacy** [comment]
> > I also noticed this through testing (the text labels by the toggle): image
> > I'm not sure if this was an issue before but I'm gonna look into it.
> 
> so leah and i were talking about this, the way its implemented now, its not a quick fix to just invert the order to label then toggle. i spent some time trying to work with the css of it and it wasn't that helpful. leah said she'll be able to work on it tonight and also she'll try to align to the center of the page more.

I'm getting this working, I'll let you know when I push it up

**Review by sderacy — APPROVED** [review]
LGTM now - let me know what you think of the changes.

**Comment by sderacy** [comment]
@thadakn1 @nishatreddy I don't know how I feel about having these underlined especially because we don't do it anywhere else in the app: 
image


@paytonshaltis @matyis1 @lelekaz you can let me know what you think but I personally vote to not have them.

**Comment by lelekaz** [comment]
> @thadakn1 @nishatreddy I don't know how I feel about having these underlined especially because we don't do it anywhere else in the app: image
> 
> @paytonshaltis @matyis1 @lelekaz you can let me know what you think but I personally vote to not have them.

Yeah I think without looks good to me too
